features/step_definitions/trema_steps.rb

```diff
@@ -0,0 +1,62 @@
+Given(/^I use OpenFlow 1\.0$/) do
+  @open_flow_version = :open_flow10
+end
+
+Given(/^I use OpenFlow 1\.3$/) do
+  @open_flow_version = :open_flow13
+end
+
+Given(/^a socket directory named "([^"]*)"$/) do |socket_directory|
+  step %(a directory named "#{socket_directory}")
+  ENV['TREMA_SOCKET_DIR'] = socket_directory
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.